Pilot_Plant_Dataset_Doc.ipynb

```diff
@@ -0,0 +1,2231 @@
+{
```

Comment:
<p>Academic Hub datasets are structured around assets. Here assets are pieces of equipment found in a <strong><em>Glycol Plant</em></strong>: many<strong><em> feed, product and chemical tanks, heat exchangers,</em></strong> and other related equipment. Each asset has an Asset Identifier (Asset Id) and a description. This information is available through the Hub Python library (<code>ocs_academic_hub</code>) and is returned as Pandas dataframe. </p><p>Each asset is endowed with a default data view, i.e. a structure which maps all the data streams associated to the asset onto a column in a Pandas dataframe. Each data view has a Data view Id and exists within an OCS namespace, i.e. a named partition with its own Namespace ID. To request data for an asset, the following information should be provided: </p><ul><li>Namespace Id</li><li>Data view Id</li><li>Start index (timestamp)</li><li>End index (timestamp).</li><li>Interpolation interval</li></ul><p>Reference for the start/end index and interpolation interval can be found at <a href="https://ocs-docs.osisoft.com/Content_Portal/Documentation/DataViews/GetDataViewData/Quick_Start_Get_Data_View_Data.html#index" target="_blank">https://ocs-docs.osisoft.com/Content_Portal/Documentation/DataViews/GetDataViewData/Quick_Start_Get_Data_View_Data.html#index</a>.  </p><p>For specific use-cases it is possible to define specialized dataviews having only a subset of the default <strong><em>data view</em></strong> columns. Specialized <strong><em>data views</em></strong> avoid <strong><em>unnecessarily large extraneous data retrieval requests</em></strong> and reduce the time to get results<strong><em> back,</em></strong> especially with large tables with <strong><em>thousands</em></strong> of rows. Most data views are for a single asset, but <strong><em>it is</em></strong> possible to define a data view spanning multiple assets. This <strong><em>avoids</em></strong> requesting multiple single-asset data views and <strong><em>stitching</em></strong> the result into a single Pandas dataframe for analysis. </p><p>Asset metadata is static information about the asset, e.g. the equipment manufacturer, installation date, etc.  </p><p><strong>NOTE: this dataset has no multi-asset data </strong><em>views.</em> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/academic-hub/datasets/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
done